WordPress/src/main/res/layout/reader_recommended_blog_item.xml

```diff
@@ -6,6 +6,19 @@
     android:layout_height="wrap_content"
     android:background="?attr/selectableItemBackground">
 
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/site_follow_icon"
```

Comment:
Wondering if we can reuse `ReaderFollowButton` with `wpShowFollowButtonCaption` style here 🤔.